src/routes/docs/configuring/tags/+page.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Use tags to organize files on Windows
+---
+
+Tags are a way to organize your files and folders. You can assign one or more tags to any file or folder, and then use the tags to filter and sort your files. You can also use tags to search for your files.
+	
+## How to assign tags
+
+To assign a tag to a file or folder, follow these steps:
+
+- Right click the file or folder you want to tag.
+- Click on the "edit tags" menu that appears.
+- Select a tag from the list.
+
+You can also assign tags by dragging and dropping files or folders onto existing tags in the sidebar.
+
+## How to view and filter by tags
+
+To view and filter your files by tags, follow these steps:
+
+- You will see a list of your tags on the bottom of the left-hand sidebar
+- Tap on a tag to see all the files or folders with that tag.
+
+You can also click tags in the details layout to start a search. To do that, follow these steps:
```

Comment:
Is it the "details layout" or rather "Details-Layout"? What do you think @yaira2 ?

Reply:
I usally refer to it as the way it is but I don't have a strong preference.